test/run

```diff
@@ -177,6 +169,41 @@ app_cleanup() {
   fi
 }
 
+# Positive test & non-zero exit status = ERROR.
+# Negative test & zero exit status = ERROR.
+# Tests with '-should-fail-' in their name should fail during a build,
+# expecting non-zero exit status.
+evaluate_build_result() {
```

Comment:
Copy paste from s2i-python-container.

Think about to move it to 'test-lib.sh' in future. Just nit-pick